index.bs

```diff
@@ -2566,6 +2715,14 @@ To <dfn>process an attribution source</dfn> given an [=attribution source=] |sou
     [=obtaining a randomized source response=] with |randomizedResponseConfig| and |epsilon|.
 1. Set |source|'s [=attribution source/randomized trigger rate=] to the result of
     [=obtaining a randomized source response pick rate=] with |randomizedResponseConfig| and |epsilon|.
+1. Let |states| be the [=obtain a set of possible trigger states|number of possible trigger states=] with |randomizedResponseConfig|.
+1. Let |maxEventStates| be |source|'s [=attribution source/max event states=].
+1. If |maxEventStates| is equal to 0 or |maxEventStates| is greater than [=max trigger-state cardinality=]:
```

Comment:
Is this actually possible? Isn't it already checked in the "parse max event states" algorithm?

Reply:
Nice catch. Done.